packages/relay/src/index.ts

```diff
@@ -3,7 +3,9 @@ import path from 'path'
 import fs from 'fs'
 import express from 'express'
 import { ethers } from 'ethers'
-import { SQLiteConnector } from 'anondb/node.js'
+import { SQLiteConnector, PostgresConnector } from 'anondb/node.js'
```

Comment:
why do we need 2 types of DB?

Reply:
umm, @NOOMA-42  this part is automatic merge from main, but I have no idea why there's this difference between the two file.

Reply:
Yes this is for local and stage DB connection 
- Local => SQLiteConnector (memory DB)
- Stage => PostgresConnector (remote DB)